frontend/src/lib/components/general/HelpButton.svelte

```diff
@@ -1,29 +1,28 @@
 <script lang="ts">
 	import { browser } from '$app/environment';
 	import { clickOutside } from '$lib/util/clickOutside';
-	import { mdiHelp } from '@mdi/js';
-	import IconButton, { Icon } from '@smui/icon-button';
+	import { tooltip } from '$lib/util/tooltip';
+	import { Icon } from '@smui/icon-button';
 	import Paper, { Content } from '@smui/paper';
 	import { fade } from 'svelte/transition';
 
-	export let docsLink = 'https://zenoml.com/docs/intro';
-	export let optionsBelow = false;
+	export let showOptions = false;
 
-	let showOptions = false;
+	let docsLink = 'https://zenoml.com/docs/intro';
```

Comment:
Used this as export to deep-link to different parts of the docs previously but probably not necessary anymore.